org.palladiosimulator.analyzer.slingshot.behavior.spd/src/org/palladiosimulator/analyzer/slingshot/behavior/spd/interpreter/ScalingTriggerInterpreter.java

```diff
@@ -105,7 +112,7 @@ public InterpretationResult addFrom(final InterpretationResult other, final Logi
 						if (temp instanceof final LogicalANDComboundFilter chain) {
 							chain.add(other.triggerChecker);
 						} else {
-							final LogicalANDComboundFilter comboundFilter = new LogicalANDComboundFilter();
+							final LogicalANDComboundFilter comboundFilter = new LogicalANDComboundFilter(null);
```

Comment:
No, these nulls were autogenerated, since each Filter technically has to hold a state.

However, in case of a combound filter, this is actually not needed, since the "child" filters already have the state. I removed the constructor completely.